File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/RefreshTokenRequest.java
package com.diagorn.sparkathon.auth.dto.auth;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * Refresh token request
 *
 * @author diagorn
 */
@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class RefreshTokenRequest {
    /**
     * Refresh token
     */
    @Schema(description = "Refresh token", example = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJkaWFnb3JuIiwiaWF0IjoxNzU1ODc1MjAzLCJleHAiOjE3NTU4ODM4NDN9.3y8Jbs4Zz0tx7j0u7ZuIbppJASYt1kIl610bMnN7Zuk")
    private String refreshToken;
}

File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/auth/RefreshTokenRequest.kt
package com.diagorn.sparkathon.auth.dto.auth

import io.swagger.v3.oas.annotations.media.Schema

class RefreshTokenRequest(
    /**
     * Refresh token
     */
    @Schema(
        description = "Refresh token",
        example = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJkaWFnb3JuIiwiaWF0IjoxNzU1ODc1MjAzLCJleHAiOjE3NTU4ODM4NDN9.3y8Jbs4Zz0tx7j0u7ZuIbppJASYt1kIl610bMnN7Zuk"
    )
    var refreshToken: String
) {
}